Library/Homebrew/cask/lib/hbc/cli/style.rb

```diff
@@ -11,7 +11,7 @@ def self.help
 
       def run
         install_rubocop
-        system "rubocop", *rubocop_args, "--", *cask_paths
+        system({ "XDG_CACHE_HOME" => HOMEBREW_CACHE }, "rubocop", *rubocop_args, "--", *cask_paths)
```

Comment:
You end up with a directory like this though that I don't think is ideal:
```
/Users/mike/Library/Caches/Homebrew/501/rubocop_cache/
```

Why 501?